tests/unit_tests/device_setup_plans/test_utils.py

```diff
@@ -37,3 +48,48 @@ def my_plan():
     eiger.async_stage.assert_called_once()
 
     eiger.disarm_detector.assert_called_once()
+
+
+@pytest.fixture()
+def null_plan():
+    yield from bps.null()
```

Comment:
Should: This isn't doing what it looks like. Because you can create fixtures with generators this is actually returning a `Msg` so when you use it in the plans below you're passing a `Msg` in to `start_preparing_data_collection_then_do_plan` rather than a generator. This isn't currently causing any errors because you're throwing an exception before hitting that but it might cause confusion down the line. You could fix it here with:
```
@pytest.fixture()
def null_plan():
    return bps.null()
```

Reply:
well spotted!